src/Blockcore.Indexer.Core/Storage/Mongo/MongoData.cs

```diff
@@ -890,6 +909,30 @@ protected virtual async Task OnDeleteBlockAsync(SyncBlockInfo block)
          await Task.CompletedTask;
       }
 
+      public QueryResult<QueryMempoolTransaction> GetMemoryTransactionsSlim(int offset, int limit)
```

Comment:
If all we return is the list of ids it would be much easier to understand that if the name reflected that

Reply:
like `QueryMempoolTransactionIds` ?

Reply:
ok renamed to `QueryMempoolTransactionHashes`